src/forms/FormLines.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { Meta } from "@storybook/react";
+import { FieldGroup, FormLines } from "src/forms/FormLines";
+import { TextField } from "src/inputs/TextField";
+
+export default {
+  component: FormLines,
+  title: "Forms/Form Lines",
+} as Meta;
+
+export function FlatList() {
+  return (
+    <FormLines>
+      <TextField label={"First"} value={"first"} onChange={() => {}} />
+      <TextField label={"Last"} value={"last"} onChange={() => {}} />
+    </FormLines>
+  );
+}
+
+export function SmallFlatList() {
+  return (
+    <FormLines width="sm">
+      <TextField label={"First"} value={"first"} onChange={() => {}} />
+      <TextField label={"Last"} value={"last"} onChange={() => {}} />
+    </FormLines>
+  );
+}
+
+export function SideBySide() {
+  return (
+    <FormLines>
+      <FieldGroup>
```

Comment:
**Thoughts**: Any thought on making it `FieldLines` of `formGroup` to keep the names consistent?